.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+<!--
+Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
+Any pull request must pass the automated Travis tests.
+-->
```

Comment:
"automated Travis tests" and if applicable code style checks and be covered by tests.